### PR TITLE
Replaced incorrect code example.

### LINKED ISSUE #1415: Code example for topic 'Classes and Structs' is wrong
On the following page there is a section called 'Example' that is wrong:
https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/index.md

The text of the topic says the following:

>Example
>In the following example, MyCustomClass is defined with three members at the top level of the ProgrammingGuide namespace. An instance (object) of MyCustomClass is created in the Main method in the Program class, and the object’s methods and properties are accessed by using dot notation. 
>[!code-cscsProgGuideObjects#88] 

The code sample that is linked to, however, is the following:
https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/index_1.cs

This code snippet shows the IEquatable<T> interface and appears to have nothing to do with the topic.

I would try to fix this and submit a pull request but I'm not sure how to fix the code sample link.

### PR DESCRIPTION
**Pull request by rpetrusha**
# Replaced incorrect code example

## Summary
An incorrect code example was imported by migration. This PR replaces it with a modified version of the correct example.

Fixes #1415 

## Suggested Reviewers

@mairaw @billwagner 

/cc @Treit

### CODE REVIEW

**Comment by mairaw** [comment]
Should we move this to the correct snippet location already?

**Comment by rpetrusha** [comment]
@mairaw I've already put it in the correct snippet location.

**Comment by mairaw** [comment]
Oh yeah! Totally my bad.

File: docs/csharp/programming-guide/classes-and-structs/index.md [hunk 1]
```diff
@@ -52,9 +51,9 @@ Classes and structs are two of the basic constructs of the common type system in
  For more information, see [Classes](../../../csharp/programming-guide/classes-and-structs/classes.md), [Objects](../../../csharp/programming-guide/classes-and-structs/objects.md), and [Structs](../../../csharp/programming-guide/classes-and-structs/structs.md).  
   
 ## Example  
- In the following example, `MyCustomClass` is defined with three members at the top level of the `ProgrammingGuide` namespace. An instance (object) of `MyCustomClass` is created in the `Main` method in the `Program` class, and the object’s methods and properties are accessed by using dot notation.  
+ In the following example, `customClass` in the `ProgrammingGuide` namespace has three members: an instance constructor, a property named `Number`, and a method named `Multiply`. The `Msin` method in the `Program` class creates an instance (object) of `myCustomClass`, and the object’s method and property are accessed by using dot notation.
```
Comment (mairaw) [inline docs/csharp/programming-guide/classes-and-structs/index.md]: the name of the class in your example is CustomClass with capital C

Comment (mairaw) [inline docs/csharp/programming-guide/classes-and-structs/index.md]: also, a typo: Msin should be Main and myCustomClass should be just CustomClass?

in addition to that: dot notation -> the dot notation?

File: samples/snippets/csharp/programming-guide/classes-and-structs/class1.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          Number = 0;
+      }
+   }
+
+   // Another class definition that contains Main, the program entry point.
+   class Program
+   {
+      static void Main(string[] args)
+      {
+         // Create an object of type MyCustomClass.
```
Comment (mairaw) [inline samples/snippets/csharp/programming-guide/classes-and-structs/class1.cs]: MyCustomClass -> CustomClass

File: docs/csharp/programming-guide/classes-and-structs/index.md (hunk 1 above)
Comment (rpetrusha) [inline docs/csharp/programming-guide/classes-and-structs/index.md]: I've fixed these except for "the dot notation" -- simply "dot notation" seems to me, at least, to be less awkward.

**Comment by rpetrusha** [comment]
@mairaw Thanks for catching these, Maira. I've fixed them.

**Comment by mairaw** [comment]
LGTM. :shipit:

**Pull request merged**